configuration/init/systemd/tedge-mapper-collectd.service

```diff
@@ -1,9 +1,12 @@
 [Unit]
 Description=tedge-mapper-collectd converts Thin Edge JSON measurements to Cumulocity JSON format.
 After=syslog.target network.target mosquitto.service
+ConditionPathExists=|/etc/tedge/mosquitto-conf/c8y-bridge.conf
+ConditionPathExists=|/etc/tedge/mosquitto-conf/az-bridge.conf
 
 [Service]
 User=tedge
+ExecStartPre=/bin/bash -c '/usr/bin/systemctl is-active --quiet collectd.service'
```

Comment:
It is better to use the `Requires` field to list the hard dependency to the `collectd.service` instead of using the `ExecStartPre`.

Checkout the [systemd docs](https://www.freedesktop.org/software/systemd/man/systemd.service.html) for more info